### ISSUE TITLE
Connect a printer through domain name

### ISSUE THREAD

**Issue by Benigans** [dunkmann00/PrusaLinkiOS#6]
Hi !

First, thanks a lot for your app ! So useful for us on iOS !

Usually, i access my printer through its domain name (ex: printer.domain.com).
In the app, I can only put IP address.
Do you think you can add the possibility of using the domain name of the printer ?

**Comment by dunkmann00** [comment]
Hello and thank you!

So the main reason the printer location is limited to IP address is because it makes it easy to tell if the location is a local device. This is relevant because one of the questions asked for the App Store review is "**Does your app contain unrestricted web access, such as with an embedded browser?**" Because its limited to only local, the answer to that is no.

This question has to do with the age rating the app gets. I'm not sure what would happen if I changed it to yes, but since I genuinely had no intention of giving people a web browser to use as a general web browser, it seemed appropriate to limit its use. Also, I have a MK4, and Prusa only seems to intend for people to connect to that (and the MINI and XL for that matter) with an IP Address, at least for now anyway. I wasn't really thinking about the Raspberry Pi PrusaLink users.

So thats just to give you an idea of why it is the way it is. But let me ask you, is the printer that you are trying to connect to on your local network? And is it running PrusaLink through a RaspberryPi?

**Comment by Benigans** [comment]
Ok I understand.
I have a Prusa Mini on the last beta firmware. No rpi.
I mainly use domain name because when accessing PrusaLink through a VPN when I'm outside it's easier for me to use domain name (I had to set a 1:1 NAT so the IP address of the Prusa is not the same locally than through the VPN). 

But in your last update you add the possibility to save multiple printers. I just set a "second" printer with my printer's VPN address and that's ok for me.

My case is quite an edge case.
